packages/webpack-cli/lib/webpack-cli.js

```diff
@@ -1023,7 +1023,7 @@ class WebpackCLI {
                     },
                     formatHelp: (command, helper) => {
                         const termWidth = helper.padWidth(command, helper);
-                        const helpWidth = helper.helpWidth || 80;
+                        const helpWidth = helper.helpWidth || process.env.WEBPACK_CLI_HELP_WIDTH || 80;
```

Comment:
For testing purpose and for edge case where developer can set width of TTY

Reply:
https://github.com/webpack/webpack.js.org/pull/4900